client/tso_stream.go

```diff
@@ -139,7 +139,7 @@ func (s *pdTSOStream) processRequests(
 		}
 		return
 	}
-	tsoBatchSendLatency.Observe(float64(time.Since(batchStartTime)))
+	tsoBatchSendLatency.Observe(time.Since(batchStartTime).Seconds())
```

Comment:
If the time delta is less than 1 second, will we got zero as the value of Observe?Is it expected?